R/tm_g_association.R

```diff
@@ -1,4 +1,4 @@
-#' Stack plots of variables and show association with reference variable
+#' teal Module: Stack Plot with Association
```

Comment:
Do you think that association alone is enough to describe this module?

Consider reverting to old _(long)_ title

```suggestion
#' teal Module: Stack Plots of Variables and Show Association with Reference Variable
```

Reply:
Thanks! I changed it back now.